src/components/common/Extensions/SpecificSidePanel/SpecificSidePanel.tsx

```diff
@@ -44,11 +45,11 @@ const SpecificSidePanel: FC<SpecificSidePanelProps> = ({ extensionData }) => {
               </div>
               <div className="flex flex-col justify-start gap-y-2 md:flex-row md:flex-wrap">
                 <div className="flex flex-wrap gap-1">
-                  {statuses.map((status) => (
+                  {statuses.map((status, index) => (
                     <ExtensionStatusBadge
                       key={status}
                       mode={status}
-                      text={status}
+                      text={badgeMessages[index]}
```

Comment:
I feel like it would be safer to just have a map of badge messages instead of using a state so like:


```ts
type IoannaStatus = 'installed' | 'enabled' | 'disabled' | 'deprecated';

const badgeMessages: Record<IoannaStatus, MessageDescriptor> = {
  // keys will be auto-completed at this point
  installed: {
    id: 'status.installed',
  },
  disabled: {
    id: 'status.disabled'
  },
  ...
}

const [statuses, setStatuses] = useState<IoannaStatus[]>([]);

// Update the statuses state, without worrying about the order:
setStatuses(state => [...state, 'disabled']);

// Then finally just
{statuses.map(status => (
  <ExtensionStatusBadge
    key={status}
    mode={status}
    text={formatMessage(badgeMessages[status])}
  />
)}
```